sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -3471,7 +3471,7 @@
                   "properties": {},
                   "hideLabel": true,
                   "customClass": "formio-panel-unset",
-                  "customConditional": "show = data.disbursement.disbursement1MSFAANumber !== data.disbursement.disbursement2MSFAANumber && !!data.disbursement.disbursement2MSFAANumber"
+                  "customConditional": "show = data.disbursement.disbursement1MSFAAId !== data.disbursement.disbursement2MSFAAId && !!data.disbursement.disbursement2MSFAANumber"
```

Comment:
Since the condition was changed to use the id I would prefer to change `disbursement2MSFAANumber` by `disbursement2MSFAAId` also.